index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"/>
+    <title>Grow Megha Portfolio</title>
+  </head>
+
+  <body>
+    <header>
+      <nav class="header">
+        <ul class="nav-items">
+          <li class="logo"><a>Megha</a></li>
+          <li class="menu"><i class="fa-solid fa-bars"></i></li>
+        </ul>
```

Comment:
- [ ] Please change the ul and li tags here to divs. I think you don't need to use ul tags here. 
- [ ] Please move the logo link out of the nav tag.